models/jira__daily_issue_field_history.sql

```diff
@@ -15,7 +15,7 @@
 
 {% if is_incremental() %}
 -- set max date_day with lookback as a variable for multiple uses
-{% set max_date_day = jira.jira_lookback(from_date='max(date_day)', datepart='day', interval=var('lookback_window', 3)) %}
+{% set max_date_week = jira.jira_lookback(from_date='max(date_week)', datepart='week', interval=var('lookback_window', 1)) %}
```

Comment:
`date_week` is the new field we are leveraging for the incremental partition logic. However, we don't mention this field in the CHANGELOG. Would you be able to ensure this new field is documented in the yml and included in the CHANGELOG.

Reply:
Ahh thanks. Updated and regenned docs.